rest_groups.cpp

```diff
@@ -1773,6 +1772,10 @@ bool DeRestPluginPrivate::groupToMap(const ApiRequest &req, const Group *group,
         else if (item->descriptor().suffix == RAttrClass) { map["class"] = item->toString(); }
         else if (item->descriptor().suffix == RAttrUniqueId) { map["uniqueid"] = item->toString(); }
     }
+    if (map["type"] == "LightGroup")

```

Comment:
When I read the Hue docs correctly when `class` is only meant for rooms, since there are more types than Room and LightGroup it might be easier to make the condition:

`if (map["type"] != QLatin1String("Room"))`

Please also prefer using QLatin1String, which prevents creating a QString object for static C strings.

http://doc.qt.io/qt-5/qlatin1string.html#details

http://doc.qt.io/qt-5/qstring.html#operator-eq-eq-1

Reply:
Indeed, I'll change that. Thanks for the pointer to QLatin1String, I'm not too familiar with QT.